src/main/java/com/blackducksoftware/integration/hub/alert/channel/email/template/MimeMultipartBuilder.java

```diff
@@ -72,10 +72,6 @@ public void addEmbeddedImages(final Map<String, String> contentIdsToFilePaths) {
         this.contentIdsToFilePaths.putAll(contentIdsToFilePaths);
     }
 
-    public void addAttachments(final List<String> attachmentFilePaths) {
```

Comment:
We actually want to keep this because a future feature will be attaching a CSV file to emails.